assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -541,12 +544,38 @@ $card-preview-zoom-bottom-offset:       6rem;
 
 .cart-additionalCheckoutButtons {
     @extend %additionalCheckoutButtons;
+    
+    @media (max-width: $screen-small) {
+        display: flex;
+        flex-direction: column;
+
+        >div {
+            width: 100%;
+
+            >p {
+                text-align: center !important;
+            }
+        }
+    }
 }
 
 .previewCart-additionalCheckoutButtons {
-    @extend %additionalCheckoutButtons;
-    padding-right: spacing('single');
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
     padding-bottom: spacing('single');
+
+    p {
+        text-align: center;
+    }
```

Comment:
It is better to add some classes

Reply:
I agree with you, but I took the original template "%additionalCheckoutButtons" for styling buttons as an example. Do you think I should change the tags to classes anyway?
<img width="495" alt="465_tags_style" src="https://user-images.githubusercontent.com/82589781/120831168-f35a0b00-c567-11eb-9425-1d041fec502d.png">

Reply:
Do we have scenarios when other layout can be rendered in our case?